### DISCUSSION TITLE
Standalone Mode & globals

### DISCUSSION THREAD

**Discussion by adsTestin** [webdriverio/webdriverio#10674] [Q&A]
Hello, 
How can I enable or utilize the WebDriverIO globals (browser, $, $$) in Standalone mode?

Best regards

**Comment by christian-bromann** [comment]
@adsTestin you have to register them yourself, e.g.

```js
globalThis.browser = browser
```

**Reply by adsTestin** [reply]
Thanks, but it's not as simple as that.

How to extend globalThis to recognize the browser!

**Reply by christian-bromann** [reply]
When you say "extend `globalThis` I assume you are using TypeScript. There are a lot of TypeScript tutorials on the internet that help you with that. I also always have to google for this 😉

**Reply by adsTestin** [reply]
Yes there is many, but no one works in practice. 

Thank you and if there is no wdio solution, we continue using cucumberjs context! 

Have a nice day!
